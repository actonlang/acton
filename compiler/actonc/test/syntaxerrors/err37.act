# Binary literal error
def bad_binary():
    return 0b102