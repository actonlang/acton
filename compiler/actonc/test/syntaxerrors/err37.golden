Building file test/syntaxerrors/err37.act using temporary scratch directory
[error Parse error]: unexpected "b1"
                     expecting ';', call arguments, slice/index expression, comma, digit, end of line, if clause, or operator

     +--> test/syntaxerrors/err37.act@3:13-3:14
     |
   3 |     return 0b102
     :             ^ 
     :             `- unexpected "b1"
     :                expecting ';', call arguments, slice/index expression, comma, digit, end of line, if clause, or operator
     :                
-----+
